app/src/main/java/edu/card/clarity/presentation/myCardScreen/CreditCardItemUiState.kt

```diff
@@ -1,9 +1,12 @@
 package edu.card.clarity.presentation.myCardScreen
 
 import androidx.compose.ui.graphics.Color
+import java.util.UUID
 
 data class CreditCardItemUiState(
     val cardName: String,
     val dueDate: String,
-    val backgroundColor: Color
+    val isReminderEnabled: Boolean,
+    val backgroundColor: Color,
+    val id: UUID?,
```

Comment:
This ID won't be null as everything fetched from DB definitely has a primary key (You can use the not-null assertion operator if Kotlin gives you a complaint)